addons/initialise-sprite-position/userscript.js

```diff
@@ -0,0 +1,24 @@
+export default async function ({ addon }) {
+  let vm = addon.tab.traps.vm;
+  let oldAddSprite = vm.constructor.prototype.addSprite;
+  vm.constructor.prototype.addSprite = function (input) {
+    let spriteObj,
+      stringify = true;
+    alert(typeof input);
+    if (typeof input === "object") [spriteObj, stringify] = [input, false];
+    else spriteObj = JSON.parse(input);
+    let isEmpty = spriteObj.costumes?.[0]?.baseLayerMD5 === "cd21514d0531fdffb22204e0ec5ed84a.svg";
+    alert("moo");
+    if (isEmpty || !spriteObj.tags || !addon.settings.get("library")) {
+      if (spriteObj.scratchX) {
+        spriteObj.scratchX = +addon.settings.get("x");
+        spriteObj.scratchY = +addon.settings.get("y");
+      }
+      if (spriteObj.x) {
+        spriteObj.x = +addon.settings.get("x");
+        spriteObj.y = +addon.settings.get("y");
```

Comment:
No need to convert the return value from `addon.settings.get()` into a number, it already is

Reply:
It isn't, see https://discord.com/channels/806602307750985799/806605006072709130/830073698420916306

Reply:
If that doesn't return a number it's a bug in our end

Reply:
Yup, it is, fairly sure you need to add the .number modifier to the v-models to automatically typecast it to a number e.g. `v-model.number="addonSettings[addon._addonId]"` or something. I'll test it out quickly and pr.